src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -42,7 +43,7 @@ type WorkspaceMenuItem = {
     icon: IconAsset;
     action: () => void;
     brickRoadIndicator?: ValueOf<typeof CONST.BRICK_ROAD_INDICATOR_STATUS>;
-    routeName?: ValueOf<typeof SCREENS.WORKSPACE>;
+    name?: ValueOf<typeof SCREENS.WORKSPACE>;
```

Comment:
NAB: I think the original `routeName` seemed more correct to me here instead of `name`

Reply:
I wonder why I changed it 🤔

Reply:
I'd agree with you on this though.

Reply:
Resolved!